src/clojure/nrepl/middleware/session.clj

```diff
@@ -85,7 +85,10 @@
                          ^BlockingQueue queue
                          thread-factory)))
 
-(def default-executor "Delay containing the default Executor." (delay (configure-executor)))
+(defn default-executor
+  "Delay containing the default Executor."
+  []
+  (delay (configure-executor)))
```

Comment:
I think this `delay` is probably no longer needed, as it won't be evaluated right away anyways.